GetDKAN/dkan-tools
# Move Drupal frontend commands to frontend:install, use new module

## QA Steps

1. Check out branch in local copy of dkan-tools
2. Run `dktl init && dktl demo`
3. Run `dktl drush pml --filter=frontend`; confirm that `dkan_js_frontend` is enabled and `frontend` is disabled.
4. Go to the front page of the site. Confirm frontend app is working and styles look correct.
5. Go to /user/login. Confirm the page is styled correctly (like bartik).
